### PR TITLE
ENG-7206 - central configuration changes

### PR DESCRIPTION
**Pull request by muthuraj-mariappan**

### CODE REVIEW

File: deploy-operator-openshift/digitalai-deploy/kubernetes/daideploy_cr.yaml [hunk 1]
```diff
@@ -151,6 +151,10 @@ spec:
       readinessProbe:
         enabled: true
         failureThreshold: 6
+        httpGet:
```
Comment (muthuraj-mariappan) [inline deploy-operator-openshift/digitalai-deploy/kubernetes/daideploy_cr.yaml:154]: @anallathambi - why this change is required?

Comment (anallathambi) [reply deploy-operator-openshift/digitalai-deploy/kubernetes/daideploy_cr.yaml:154]: Misplaced property ,'ll revert it.

**Review by simhadriraju-digitalai — APPROVED** [review]
